api/endpoints/off_topic_channel_names/endpoints.py

```diff
@@ -0,0 +1,142 @@
+from typing import Optional, Union
+
+from fastapi import APIRouter, Depends
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Query, Session
+
+from api.core.database.models.api.bot import OffTopicChannelName
+from api.core.schemas import ErrorMessage
+from api.endpoints.dependencies.database import create_database_session
+
+otn = APIRouter(prefix="/off-topic-channel-names")
+
+
+def get_all_otn(db_session: Session) -> Query:
+    """Get a partial query object with .all()."""
+    return db_session.query(OffTopicChannelName).all()
+
+
+@otn.get(
+    "/",
+    status_code=200,
+    response_model=list[str],
```

Comment:
I think you should create a response model, and set `orm_mode` to True

Reply:
If I use a response pydantic model, it would return it as a dict with fields and their model values, basically:
```py
>>> httpx.get("http://127.0.0.1:8000/bot/off-topic-channel-names", headers=AUTH_HEADER).json()
[{'name': 'lemons-lemonade-stand'}]
```

But the required JSON response should be:
```py
httpx.get("http://127.0.0.1:8000/bot/off-topic-channel-names", headers=AUTH_HEADER).json()
['lemons-lemonade-stand']
```